### DISCUSSION TITLE
Help with migrations (SchemaDefinitionError)

### DISCUSSION THREAD

**Discussion by tech-cobber** [geldata/gel#1951] [Q&A] [answered]
### Can you help me with this error? 
`Error: ERROR: SchemaDefinitionError: specified migration name does not match the name derived from the migration contents: 'm1gqtx7inak2ygrhxqidrdle2wkjifbwt76yzuy3fn3hxwngv6rcra', expected 'm14nkjk3jldnnmuw7h6gka3d3kazcgch7uoxggfa24364vfyoudyda'`

### I'm having a project:
`edgedb_migrations/dbschema/schema.esdl` with [module](https://www.edgedb.com/docs/tutorial/createdb#sdl) definition
`edgedb_migrations/dbschema/migrations/` empty (for migrations)
`edgedb_migrations/docker-compose.yml` to run EdgeDB container

### What I'm doing is:
0) `docker-compose up -d`
1) create migrations
![image](https://user-images.githubusercontent.com/50849313/98126325-847bdd00-1ec6-11eb-9ef1-e55594af9d6f.png)
2) apply them
![image](https://user-images.githubusercontent.com/50849313/98126602-d45aa400-1ec6-11eb-832d-777168e623dc.png)
3) change few fields inside `schema.esdl`
4) create new migrations
![image](https://user-images.githubusercontent.com/50849313/98126763-ff44f800-1ec6-11eb-913a-76f958e65881.png)
5) apply new one and receive error:
![image](https://user-images.githubusercontent.com/50849313/98126852-171c7c00-1ec7-11eb-9b32-de1039cb18a2.png)

### 00001.edgeql:
![image](https://user-images.githubusercontent.com/50849313/98127039-4af7a180-1ec7-11eb-9e10-689bf7564112.png)
### 00002.edgeql:
![image](https://user-images.githubusercontent.com/50849313/98127145-695d9d00-1ec7-11eb-9994-c3d7bc0080b2.png)

Is it a bug or I'm doing something wrong. Can you please help me?

**Comment by elprans** [comment, marked as answer]
This is a bug, which was fixed in #1935 and #1929

**Reply by tech-cobber** [reply]
Got it, thank you
